tsconfig.json

```diff
@@ -1,24 +1,23 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ESNext",
+    "useDefineForClassFields": true,
     "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
+      "DOM",
```

Comment:
@aulneau updates on tsconfig for micro-stacks libraries or improvements?

Reply:
this was something I was testing when I had issues running the app -- it should make no difference, it's the default tsconfig from the react-ts vite starter (which we should use if you want to work with my other PR) :)